src/twist_mux_diagnostics.cpp

```diff
@@ -84,10 +89,7 @@ void TwistMuxDiagnostics::diagnostics(diagnostic_updater::DiagnosticStatusWrappe
               static_cast<int>(lock_h.getPriority()));
   }
 
-  stat.add("current priority", static_cast<int>(status_.priority));
-
-  stat.add("loop time in [sec]", status_.main_loop_time);
-  stat.add("data age in [sec]", status_.reading_age);
+  stat.add("current lock priority", static_cast<int>(status_.priority));
```

Comment:
@ipa-fxm `current lock priority` replaces `current priority`.

The others are simply not used anywhere.

The `current lock priority` is related with the locks, not the topics. But I honestly didn't have time to look if I can make the diagnostics more clear.

Do you mind if I merge this and then we discuss how to improve it on an issue?
I'd like to have this merged for ROS kinetic release.